pandas/tests/tools/test_to_numeric.py

```diff
@@ -627,3 +627,9 @@ def test_non_coerce_uint64_conflict(errors, exp):
     else:
         result = to_numeric(ser, errors=errors)
         tm.assert_series_equal(result, ser)
+
+
+def test_failure_to_convert_uint64_string_to_NaN():
+    # GH 32394
+    result = to_numeric("uint64", errors="coerce")
+    assert np.isnan(result)
```

Comment:
I think this is a good test, but we generally add the OP example as a test too.

Reply:
This is basically the OP example, you know, since the first thing that `to_numeric` does is putting that scalar value in a object array (the bug was about the "uint64" string, not about whether it was passed as a list, series, array or scalar)